docs/source/2018Michael.rst

```diff
@@ -113,14 +136,37 @@ Hurricane Track and Intensity
+
+    * Hurricane track from SRW_GFSv15p2 is closer to the Best Track compared with MRW_GFSv15p2 and MRW_GFSv16beta. 
+    * Due to lack of information contained in the initial and boundary condition files for this relatively old (2018) case, the SRW App could not run with suite ``SRW_RRFSv1alpha``.
+
+
+    .. figure:: images/2018Michael/tracker_timeseries_Michael_srwv1.png
+      :width: 1200
+      :align: center
+
+      Time series of the vortex maximum surface wind speed (WS, left panel) and minimum mean sea level pressure (MSLP, right panel)
+
+    * The maximum surface wind speed at the vortex center in SRW_GFSv15p2 (90 kts) does not reach the peak intensity identified in Best Track data (140 kts).
+    * The minimum sea level pressure from SRW_GFSv15p2 is 948 hpa, which is larger than Best Track data.
+  
+
 ......................
 Summary and Discussion
 ......................
 
-The physics compset of GFS.v16beta in MRW.v1.0 does not show improvements compared with the compset of GFS.v15p2 in simulating Hurricane Michael track and intensity. The physics for MRW_GFS.v16beta was frozen in Jan 2020 for the public release of UFS MRW App version 1.0. Several major changes have been made to the GFS model after that, which include more vertical levels, several tuning parameters, and modified background diffusivity as a function of stability in the PBL scheme, etc. (`Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_). In GFS.v16.0.10 (GitHub tag created in Jun 2020), the simulated hurricane landfall location and hurricane intensity is closer to Best Track data compared to MRW_GFSv15p2.
+The GFS.v16beta compset in MRW.v1.0 does not show improvements in simulating Hurricane Michael track and intensity when compared with the GFS.v15p2 compset. The physics for MRW_GFS.v16beta was frozen in Jan 2020 for the public release of UFS MRW App version 1.0. Several major changes have been made to the GFS model after that, which include more vertical levels, several tuning parameters, and modified background diffusivity as a function of stability in the PBL scheme, etc. (`Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_). In GFS.v16.0.10 (GitHub tag created in Jun 2020), the simulated hurricane landfall location and hurricane intensity is closer to Best Track data compared to MRW_GFSv15p2.
+The results from SRW App v1.0 show better track and intensity in simulating Hurricane Michael when compared to the results from MRW App v1.0, possibly because the SRW App 1.0 has a newer code base. However, the SRW App forecasts have shorter duration because of limitations in the availability of lateral boundary conditions for longer lead times.
```

Comment:
Is the initialization date the same for MRW and SRW for Hurricane Michael?

Reply:
The initialization date is different for MRW and SRW for Hurricane Michael due to that the boundary conditions only have 90 hours data for SRW. @climbfuji

Reply:
So this means that the SRW App is initialized closer to the analysis time. This should be mentioned when saying that the SRW App produces better tracks, because the shorter the forecast time, the better the tracks will be. Same for all other hurricanes (if applicable).

Reply:
Yes, thanks!

Reply:
Yes, thanks!